src/main.js

```diff
@@ -525,6 +527,54 @@ class MiniGraphCard extends LitElement {
       </svg>`;
   }
 
+  renderGridLines() {
+    const {
+      height, hours_to_show, show,
+    } = this.config;
+    const grid_lines_type = show.grid_lines_type ? show.grid_lines_type : false;
+    const grid_lines_ratio = (show.grid_lines_ratio && Number.isInteger(show.grid_lines_ratio))
+      ? show.grid_lines_ratio
+      : 2;
+
+    const containerWidth = 500;
+    let spanInHours;
+
+    switch (grid_lines_type) {
+      case '5minute':
+        spanInHours = 1 / 12;
+        break;
+      case 'hour':
+      default:
+        spanInHours = 1;
+        break;
+      case 'day':
+        spanInHours = 24;
+        break;
+      case 'week':
+        spanInHours = 168;
+    }
+
+    let numLines = hours_to_show / spanInHours;
+    const spanFactor = Math.ceil(hours_to_show / spanInHours) / (hours_to_show / spanInHours);
+    const thickPart = containerWidth * spanFactor / Math.ceil(numLines);
+    const thinPart = thickPart / (grid_lines_ratio + 1);
+    numLines *= (grid_lines_ratio + 1);
+
+    const lines = [];
+    for (let i = 0; i < numLines - 1; i += 1) {
+      const x = containerWidth - thinPart * (i + 1);
+      // const timeLabel = hours_to_show / numLines * (i + 1);
+
+      if ((i + 1) % (grid_lines_ratio + 1) > 0) {
```

Comment:
Can we simplify this?
Untested (please check if it OK):
```
      let stroke;
      if ((i + 1) % (grid_lines_ratio + 1) > 0) {
        stroke = "var(--mcg-grid-line-thin-color, var(--divider-color))";
      } else {
        stroke = "var(--mcg-grid-line-thick-color, rgb(from var(--divider-color) R G B /0.5))";
      }
      lines.push(svg`<line x1=${x} y1="0" x2=${x} y2=${height} stroke="${stroke}" stroke-width="0.5"/>`);
```

Reply:
Seems to work fine